cuda_core/cuda/core/experimental/_memory.py

```diff
@@ -44,9 +42,13 @@ class Buffer:
     """
 
     # TODO: handle ownership? (_mr could be None)
-    __slots__ = ("_ptr", "_size", "_mr",)
+    __slots__ = (
+        "_ptr",
+        "_size",
+        "_mr",
+    )
```

Comment:
I don't understand why this `__slots__` would be broken into multiple lines... This doesn't happen in other places?

Reply:
yeah I agree that is weird, I will look into this before submitting anything